shardingsphere-proxy/shardingsphere-proxy-backend/src/test/java/org/apache/shardingsphere/proxy/backend/text/distsql/ral/common/updatable/ImportSchemaConfigurationHandlerTest.java

```diff
@@ -0,0 +1,153 @@
+        when(shardingSphereMetaData.getResource().getDataSources()).thenReturn(createDataSourceMap());
+        when(shardingSphereMetaData.getRuleMetaData().getConfigurations()).thenReturn(Collections.singletonList(createShardingRuleConfiguration()));
+        when(contextManager.getMetaDataContexts().getMetaData(schemaName)).thenReturn(shardingSphereMetaData);
+        ProxyContext.getInstance().init(contextManager);
+    }
+    
+    @Test
+    public void assertImportSchemaExecutor() throws SQLException {
+        Map<String, DataSource> dataSourceMap = ProxyContext.getInstance().getContextManager().getDataSourceMap(schemaName);
+        assertNotNull(dataSourceMap);
+        Collection<RuleConfiguration> ruleConfigurations = ProxyContext.getInstance().getContextManager().getMetaDataContexts().getMetaData(schemaName).getRuleMetaData().getConfigurations();
+        assertNotNull(ruleConfigurations);
+        ImportSchemaConfigurationHandler handler = new ImportSchemaConfigurationHandler().init(getParameter(createSqlStatement(), mockConnectionSession()));
+        ResponseHeader responseHeader = handler.execute();
+        assertTrue(responseHeader instanceof UpdateResponseHeader);
+    }
+    
+    private ShardingRuleConfiguration createShardingRuleConfiguration() {
+        ShardingRuleConfiguration result = new ShardingRuleConfiguration();
+        result.getTables().add(createTableRuleConfiguration());
+        result.setDefaultDatabaseShardingStrategy(new StandardShardingStrategyConfiguration("order_id", "ds_inline"));
+        result.setDefaultTableShardingStrategy(new NoneShardingStrategyConfiguration());
+        result.getKeyGenerators().put("snowflake", new ShardingSphereAlgorithmConfiguration("SNOWFLAKE", new Properties()));
+        Properties props = new Properties();
+        props.setProperty("algorithm-expression", "ds_${order_id % 2}");
+        result.getShardingAlgorithms().put("ds_inline", new ShardingSphereAlgorithmConfiguration("INLINE", props));
+        result.setScalingName(scalingName);
+        result.getScaling().put(scalingName, null);
+        return result;
+    }
+    
+    private Map<String, DataSource> createDataSourceMap() {
+        Map<String, DataSource> result = new LinkedHashMap<>(2, 1);
+        result.put("ds_0", createDataSource("demo_ds_0"));
+        result.put("ds_1", createDataSource("demo_ds_1"));
+        return result;
+    }
+    
+    private DataSource createDataSource(final String dbName) {
+        HikariDataSource result = new HikariDataSource();
+        result.setJdbcUrl(String.format("jdbc:mysql://127.0.0.1:3306/%s?serverTimezone=UTC&useSSL=false", dbName));
+        result.setUsername("root");
+        result.setPassword("");
+        result.setConnectionTimeout(30000L);
+        result.setIdleTimeout(60000L);
+        result.setMaxLifetime(1800000L);
+        result.setMaximumPoolSize(50);
+        result.setMinimumIdle(1);
+        return result;
+    }
```

Comment:
@yx9o Using h2 database could solve the exception of ci, please refer https://github.com/apache/shardingsphere/pull/16147/files#diff-48ebddcde35067de8df4e9f553c535a82936f1da029378744ab1d6ced60a502c

Reply:
> @yx9o Using h2 database could solve the exception of ci, please refer https://github.com/apache/shardingsphere/pull/16147/files#diff-48ebddcde35067de8df4e9f553c535a82936f1da029378744ab1d6ced60a502c

Thank you.